content/applications/hr/employees/new_employee.rst

```diff
@@ -80,7 +63,7 @@ Resumé
 ~~~~~~
 
 Next, the employee's work history is entered in the :guilabel:`Resumé` tab. Each previous experience
-must be entered individually. Click :guilabel:`Create a New Entry`, and the :guilabel:`Create Resumé
+must be entered individually. Click :guilabel:`Create a new entry`, and the :guilabel:`Create Resumé
```

Comment:
This is actually in lower case in the new Milk version. I was also shocked since they are not typically written like this- not sure if this ill be the way they are doing the UI now, but this is correct for the latest V16 version (it could always change back to title case in the next rev, in which case, I'll change it back again =).